From: Alberto Garcia <berto@igalia.com>
Description: Don't install the fuse.desktop file
 We handle this in debian/rules because we need two separate versions
 for GTK+ and SDL.
Forwarded: not-needed
Index: fuse-emulator/data/Makefile.am
===================================================================
--- fuse-emulator.orig/data/Makefile.am
+++ fuse-emulator/data/Makefile.am
@@ -25,8 +25,7 @@ if DESKTOP_INTEGRATION
 fusemimedir = $(DESKTOP_DATADIR)/mime/packages
 fusemime_DATA = data/fuse.xml
 
-appdatadir = $(DESKTOP_DATADIR)/applications
-appdata_DATA = data/fuse.desktop
+noinst_DATA = data/fuse.desktop
 
 iconsdatadir = $(DESKTOP_DATADIR)/icons/hicolor
 icons16dir = $(iconsdatadir)/16x16/apps
